common/path.go

```diff
@@ -39,6 +40,7 @@ func FileExist(filePath string) bool {
 	return true
 }
 
+// AbsolutePath returns Datadir + filename, or filename if it is absolute.
 func AbsolutePath(Datadir string, filename string) string {
```

Comment:
Please lowercase `Datadir` here, not sure how it got capitalized in the first place.